### PR TITLE
refactor(traffic_light_classifier): update training docs

### PR DESCRIPTION
**Pull request by kaancolak**
## Description

Training of the traffic light classifier model isn't up-to-date. Model training and ONNX conversion scripts changed up-to-date repos of mmlab.

I have updated the custom training and detailed more.

https://github.com/autowarefoundation/autoware-documentation/pull/447

## Tests performed

Not applicable.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [X] I've confirmed the [contribution guidelines].
- [X] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by kaancolak** [comment]
@yukke42 @miursh @yukkysaito Could you look into this PR?

**Comment by yukkysaito** [comment]
Autoware Documentation might be better written where it is implementation-independent.
@mitsudome-r 
In that case, would it be better to skip the link from Autoware Documentation to README.md?

**Comment by kaancolak** [comment]
> Thank you for your PR 👍 Autoware Documentation might be better written where it is implementation-independent. @mitsudome-r In that case, would it be better to skip the link from Autoware Documentation to README.md?

@yukkysaito Saito-san thanks for your feedback. Mitsudome-san also agreed with you, I will refactor PR after it is done.

**Comment by yukkysaito** [comment]
@mitsudome-r How about you?

**Comment by kaancolak** [comment]
> @mitsudome-r How about you?

@yukkysaito Saito-san, he also has the same idea as you. I updated the PR.

After @xmfcx upload **traffic_light_sample_dataset** to AWS, I will open PR to review.

**Comment by kaancolak** [comment]
@yukkysaito @miursh  @yukke42  Could you review this PR or assign an appropriate reviewer? FYI: @mitsudome-r

File: perception/traffic_light_classifier/README.md [hunk 1]
```diff
@@ -92,67 +92,254 @@ These colors and shapes are assigned to the message as follows:
...
-- [ResNet](https://openaccess.thecvf.com/content_cvpr_2016/html/He_Deep_Residual_Learning_CVPR_2016_paper.html)
-- [MobileNetV3](https://arxiv.org/abs/1905.02244)
-  ...
+```
+
+### Installation
+
+#### Prerequisites
+
+**Step 1.** Download and install Miniconda from the [offical website](https://mmpretrain.readthedocs.io/en/latest/get_started.html).
```
Comment (yukkysaito) [inline perception/traffic_light_classifier/README.md:150]: typo
```suggestion
**Step 1.** Download and install Miniconda from the [official website](https://mmpretrain.readthedocs.io/en/latest/get_started.html).
```

**Review by yukkysaito — APPROVED** [review]
LGTM

**Pull request merged**